build/build_all.sh

```diff
@@ -25,8 +25,6 @@ if [ "$USE_DOCKER" != "" ]; then
     filebrowser/dev \
     sh -c "\
       cd build && \
-      dos2unix build_assets.sh && \
```

Comment:
On windows, if `autocrlf` is set to `true`, git will replace LF with CRLF. So, when docker is used on windows, executing these scripts inside linux containers might fail. That's the rationale to use `dos2unix`.

Reply:
@1138-4EB you're right! Will revert those two lines 😄